docs/source/audio/datasets.rst

```diff
@@ -0,0 +1,93 @@
+Datasets
+========
+
+The `audio` collection expect the training, validation and tests datasets in either NeMo format or Lhotse format.
+
+NeMo Format
+-----------
+
+Each dataset consist of a set of utterances in individual audio files plus a manifest that describes the dataset, with information about one utterance per line (``.json``).
+There should be one manifest file per dataset that will be passed in, therefore, if the user wants separate training and validation datasets, they should also have separate manifests. Otherwise, they will be loading validation data with their training data and vice versa.
+
+
+There should be one ``JSON`` manifest file per dataset that will be passed in, therefore, if the user wants separate training and validation datasets, they should also have separate manifests. Otherwise, they will be loading validation data with their training data and vice versa. Each line of the manifest should be in the following format:
+
+In most applications, such as speech restoration or enhancement, the model will aim to transform the input audio signal into the target audio signal. In this case, each line of the manifest should have the following structure:
```

Comment:
maybe 
the model **will aim to** transform the input audio - > the model **aims to** transform the input audio